README.md

```diff
@@ -1,42 +1,75 @@
-# Helicon Python Quickstart #
+#Helicon Python Sample App
+This repository contains the files required to run the Helicon Python Quickstart.
 
-This repository contains a python publish and subscribe client example.
+Helicon is a simple, scalable, robust, code-free and generic platform to enable and productise the next generation of "online modified", real-time event stream ML/AI models.
 
-## How do I get set up? ##
+## Set up the project
 
-* Run `pip install -r requirements.txt` to install the project. 
-  To start to publish and/or subscribe to a topic stream you need to populate all the placeholder fields, defined like <placeholder>.
-  Then you can just run the `__main__` method inside each client example.
+Run `pip install -r requirements.txt` to install the project.
+To start to publish and/or subscribe to a topic stream you need to populate all the placeholder fields, defined like <placeholder>.
+Then you can just run the `__main__` method inside each client example.
 
-## Test cases ##
-
-### Publish a temperature with timestamp ###
-As part of this example, you are going to publish a json object like
+## Sample App
+### Publish
+This code is available at `helicon_publish/publish.py`. This example will publish as payload a json object like
 ```
 {
 "temperature": 26,
 "timestamp": 1632143723
 }
 ```
-defined as string `payload = f'{{"temperature": 26, "timestamp": {time.time()}}}'`. The method `time.time()` is part of the __Python 3__ code base, so you don't need to install anything else to use it.
+The method `time.time()` is part of the __Python 3__ code base, so you don't need to install anything else to use it.
+```
```

Comment:
Could you please add ```python so that it will be better displayed?

Reply:
Please, replace keycloak with authorization-server

Reply:
Should I replace all the "keycloak"s with authorization-server also in the api?

Reply:
I am doing it across the whole project